src/internet_identity/internet_identity.did

```diff
@@ -180,6 +181,33 @@ type ArchiveInfo = record {
     archive_config: opt ArchiveConfig;
 };
 
+type ActiveAnchorStatistics = record {
+    // Stats for the last completed collection period for daily and monthly active anchors
+    completed: CompletedActiveAnchorStats;
+    // ongoing periods for daily and monthly active anchors
+    ongoing: OngoingActiveAnchorStats;
+};
+
+type CompletedActiveAnchorStats = record {
+    daily_active_anchors: opt ActiveAnchorCounter;
+    monthly_active_anchors: opt ActiveAnchorCounter;
+};
+
+type OngoingActiveAnchorStats = record {
+    // Ongoing active anchor counter for
+    daily_active_anchors: ActiveAnchorCounter;
+    // Monthly active users are collected using 30 day sliding windows.
+    // This vec contains up to 29 30-day active windows each offset by one day.
+    // The vec is sorted with the first element being the next window to reach the end of the
```

Comment:
nit: this description reads a bit convoluted, maybe explain in terms of where are the newer buckets added: at the end of the vector, right?